package.json

```
@@ -27,7 +27,7 @@
     "index.js"
   ],
   "dependencies": {
-    "lodash.includes": "^4.2.0",
+    "lodash": ">= 4.17.5",
```

We want to avoid including major versions in the range

```suggestion
    "lodash": "^4.17.5",
```

will allow minor and patch updates, without doing unsafe major upgrades.
https://nodesource.com/blog/semver-a-primer